src/modules/control_allocator/ControlAllocator.cpp

```diff
@@ -285,6 +285,9 @@ ControlAllocator::Run()
 
 	perf_begin(_loop_perf);
 
+	// Push backup schedule
+	ScheduleDelayed(50_ms);
```

Comment:
Can you add the same to `ControlAllocator::init` at the end to kick off the initial backup schedule?

And rebase, which possibly already fixes CI?
> omnibus_f4sd_default.elf section `.data' will not fit in region `flash'
/opt/gcc/bin/../lib/gcc/arm-none-eabi/9.3.1/../../../../arm-none-eabi/bin/ld: region `flash' overflowed by 529 bytes

Otherwise it looks good.

Reply:
Scheduling added and rebased!